lua/vex_library/modules/server/e2helperfuncs.lua

```diff
@@ -75,4 +145,39 @@ vex.createE2Hook = function(hookname,id,callback)
             chip.context.data["vex_ran_by_"..id] = before or nil
         end)
     end)
+end
+
+
+local E2FuncNamePattern = "^([a-z][a-zA-Z0-9_]*)%("
```

Comment:
And here's the fixed pattern along the refactored functions that are now in one file, one place, much easier to maintain 😄